### ISSUE TITLE
Navigation Offline

### ISSUE THREAD

**Issue by Augustob790** [nick92/flutter_mapbox#13]
Hello, first of all, thank you very much for the plugin!!

Would it be possible to implement offline navigation? It would be a very interesting feature.

Att,

**Comment by nick92** [comment]
Yes, this would be a great feature, it is in the future pipeline but haven't had time to work on it yet
